src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -283,6 +283,11 @@ private bool IsValidSlnFilePath(string path)
             foreach (var keyEqVal in globalPropEnumerable)
             {
                 string[] keyValuePair = keyEqVal.Split("=", 2);
+                if (keyValuePair.Count() != 2)
```

Comment:
does it work for joined properties?
```
dotnet publish -p:'X=1;Y=2'
```

Reply:
Thanks, that's also a really good point. It didn't parse those properly either.